_posts/<spaceranger-23>/2020-09-04-first-reflection-<spaceranger-23>.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: "Spaceranger First Reflection!"
```

Comment:
You're missing an `author:` field.

Reply:
Thanks for making a note of the error. Resolved!